src/components/Video.tsx

```diff
@@ -19,7 +19,7 @@ export function Video({goal}: {goal: Goal}) {
   });
 
   function copyShareUrl() {
-    const goalUrl = `${API_BASE_URL}/message_preview/${goal.id}`;
```

Comment:
Wasn't sure why we were linking to the API here - we can just use our front end route for this, right?

Reply:
Cannot use the front end route. Message previews need to be server side rendered since there is not Javascript runtime in messaging apps to load a whole SPA in them. So you can see we [render the html on the backend](https://github.com/wweitzel/top90-backend/blob/9d6dd1a65a27f257575f499a2dbe625e6de6b4c0/internal/server/handlers.go#L237-L239) in order to deal with that.

Reply:
Ahh right I forgot about that. I suppose we could also set this up with React server components but that seems pretty involved. Will revert those changes.